.github/workflows/release-pkg.yml

```
@@ -79,15 +79,22 @@ jobs:
           token: ${{ secrets.PAT_GHOST }}
       - name: Update lightning version
         run: |
-          import json, os
-          fname = "versions.json"
-          with open(fname, encoding="utf-8") as fopen:
+          import json, os, re
+          tag = os.getenv('TAG')
+          fname_json = "versions.json"
+          with open(fname_json, encoding="utf-8") as fopen:
               vers = json.load(fopen)
-          vers["lightning_version"] = os.getenv('TAG')
-          with open(fname, "w", encoding="utf-8") as fopen:
+          vers["lightning_version"] = tag
+          with open(fname_json, "w", encoding="utf-8") as fopen:
               json.dump(vers, fopen, indent=2)
               # satisfy pre-commit with line fixer
               fopen.write(os.linesep)
+          fname_txt = "conda-user/user-env-requirements.txt"
+          with open(fname_txt, encoding="utf-8") as fopen:
+              reqs = fopen.read()
+          reqs = re.sub(r"lightning\w*==([\d\.]+)", f"lightning =={tag}", reqs)
```

```suggestion
          reqs = re.sub(r"lightning\w*==([\d\.]+)", f"lightning=={tag}", reqs)
```

we don't have whitespaces there